apps/accounts/models/provider_request.py

```diff
@@ -227,6 +231,9 @@ def approve(self) -> Hostingprovider:
         hp.request = self
         hp.created_by = self.created_by
 
+        # if we have approved a submission from a provider
```

Comment:
We are unarchiving all the related items, but until now, we were still leaving the provider archived. Approving the provider is an explicit acceptance of their data, so we also unarchive the provider itself now.